stripe/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -1,9 +1,9 @@
 package com.stripe.android.paymentsheet.flowcontroller
 
 import android.os.Parcelable
```

Comment:
Check there aren't any unused imports, I am seeing some in my IDE

Reply:
Are you still seeing this?

Reply:
yes, I did a pull (I am at commit: 627fdb450), and I closed and re-opened android studio.

Reply:
Which imports are unused? Ktlint isn't catching anything.

Reply:
![image](https://user-images.githubusercontent.com/77996191/110325978-0b140880-7fe6-11eb-9da5-b2f6a09d86c7.png)

Reply:
Good catch!